### DISCUSSION TITLE
Cobbler OS Installations

### DISCUSSION THREAD

**Discussion by pauliopolis** [cobbler/cobbler#3542] [Q&A]
I have been trying to get cobbler 3.3.3 working without much joy on RHEL9 from EPEL.  

For a quicker win does anyone know if 3.3.3 WILL actually work as expected on a RHEL or CentOS 8 ?

**Comment by pauliopolis** [comment]
disappointing response

**Comment by SchoolGuy** [comment]
What exactly is not working as expected? Our CI is building packages for Rocky Linux with success. In general many people in the community are using the suggested configuration.
